APIs/Dstm.ps1

```diff
@@ -25,7 +25,9 @@ class Dstm : Miner {
         $HashRate_Value = [Double]($Data.result.sol_ps | Measure-Object -Sum).Sum
         if (-not $HashRate_Value) {$HashRate_Value = [Double]($Data.result.speed_sps | Measure-Object -Sum).Sum} #ewbf fix
 
-        $HashRate | Where-Object {$HashRate_Name} | Add-Member @{$HashRate_Name = [Int64]$HashRate_Value}
+        if ($HashRate_Name -and $HashRate_Value -gt 0) {
```

Comment:
skip 0 hash rate values for more accurate reading. Sometimes the first few reding report 0, just because the miner is not ready yet.
If all was 0 then the miner would be dropped anyway